File: src/main/resources/output/IFC4X3_csharp/Entities/IfcAddress/IfcPostalAddress/IfcPostalAddress.cs
public class IfcPostalAddress : IfcAddress
{
    public IfcLabel InternalLocation { get; set; }
    public List<IfcLabel> AddressLines { get; set; }
    public IfcLabel PostalBox { get; set; }
    public IfcLabel Town { get; set; }
    public IfcLabel Region { get; set; }
    public IfcLabel PostalCode { get; set; }
    public IfcLabel Country { get; set; }

    // === WHERE CLAUSES ===
    // WR1 : EXISTS (InternalLocation) OR EXISTS (AddressLines) OR EXISTS (PostalBox) OR EXISTS (PostalCode) OR EXISTS (Town) OR EXISTS (Region) OR EXISTS (Country)
}

File: src/main/resources/output/IFC4X3_js/Entities/IfcAddress/IfcPostalAddress/IfcPostalAddress.js
class IfcPostalAddress extends IfcAddress {
    constructor() {
        /** @type {IFCLABEL} */
        this.InternalLocation = null;
        /** @type {IFCLABEL[]} */
        this.AddressLines = null;
        /** @type {IFCLABEL} */
        this.PostalBox = null;
        /** @type {IFCLABEL} */
        this.Town = null;
        /** @type {IFCLABEL} */
        this.Region = null;
        /** @type {IFCLABEL} */
        this.PostalCode = null;
        /** @type {IFCLABEL} */
        this.Country = null;
    }

    // === WHERE CLAUSES ===
    // WR1 : EXISTS (InternalLocation) OR EXISTS (AddressLines) OR EXISTS (PostalBox) OR EXISTS (PostalCode) OR EXISTS (Town) OR EXISTS (Region) OR EXISTS (Country)
}
